skycoin/skywire-services
# [WIP] Re-Implement Monitors

Fixes: #63 

Changes:
- [x] reimplement dmsg-monitor
- [x] reimplement ar-monitor
- [x] combine dmsg-monitor and ar-monitor as network-monitor
- [ ] reimplement tpd-monitor

How to test:
- run integration-env test, kill one visor, check network-monitor log
- TBD [for tpd-monitor]
